packages/beacon-node/src/node/nodejs.ts

```diff
@@ -177,19 +182,18 @@ export class BeaconNode {
       // monitoring relies on metrics data
       opts.monitoring.endpoint
     ) {
-      metrics = createMetrics(
-        opts.metrics,
-        config,
-        anchorState,
-        logger.child({module: LoggerModule.vmon}),
-        metricsRegistries
-      );
+      metrics = createMetrics(opts.metrics, anchorState.genesisTime, metricsRegistries);
       initBeaconMetrics(metrics, anchorState);
       // Since the db is instantiated before this, metrics must be injected manually afterwards
       db.setMetrics(metrics.db);
       signal.addEventListener("abort", metrics.close, {once: true});
     }
 
+    const validatorMonitor =
+      opts.metrics.enabled || opts.validatorMonitor.validatorMonitorLogs
```

Comment:
we could consider always enabling validator monitor and minimum log to debug however this might not be desired in all cases since it can be bit noisy if a lot of validators are attached so imo this is the ideal behavior, can revisit later